src/base/utils/fs.cpp

```diff
@@ -230,8 +230,14 @@ bool Utils::Fs::isValidFileSystemName(const QString &name, const bool allowSepar
 {
     if (name.isEmpty()) return false;
 
-    const QRegularExpression regex(allowSeparators ? "[:?\"*<>|]" : "[\\\\/:?\"*<>|]");
+#if defined(Q_OS_WIN)
+    const QRegularExpression regex {allowSeparators
+        ? QLatin1String("[:?\"*<>|]")
+        : QLatin1String("[\\\\/:?\"*<>|]")};
     return !name.contains(regex);
+#else
+    return (allowSeparators ? true : !name.contains('/'));
```

Comment:
IIRC, most Unix filesystems prohibit '/' and NUL characters. MacOS filesystem prohibit ':' character at classic/Carbon layer and '/' at the Unix layer (I don't know what that means).

Reply:
updated.